app/models/user_ranking.rb

```diff
@@ -1,21 +1,5 @@
 class UserRanking
-  class << self
-    def users_array
-      User.find(all.entries.collect(&:_id))
-    rescue Mongoid::Errors::DocumentNotFound
-      []
-    end
-
-    def users_hash
-      Hash[*users_array.collect { |u| [u._id, u] }.flatten]
-    end
-
-    def users
-      @users ||= users_hash
-    end
-  end
-
   def user
-    @user ||= UserRankingAllTime.users[_id]
+    @user ||= User.where(_id: _id).first
   end
```

Comment:
I guess I wrote this trying to avoid n+1 queries, are you sure we don't need it?